4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4896 @@
+{
```

Comment:
<p>Use Arviz palette, as we do in all the other NBs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>